src/llama.cpp

```diff
@@ -5517,6 +5517,11 @@ static void llm_load_vocab(
                 tokenizer_pre == "viking") {
                 vocab.type_pre = LLAMA_VOCAB_PRE_TYPE_VIKING;
                 vocab.tokenizer_clean_spaces = false;
+            } else if (
+                tokenizer_pre == "tekken") {
```

Comment:
Should this be after `"jais"` to keep the order the same as the `LLAMA_VOCAB_TYPE_PRE_*` enum?